File: for loop/cube.java
/*Calculate the cube of all numbers from 1 to a given number*/

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        int n,i;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any number:");
        n = sc.nextInt();
        
        for(i=1; i<=n;i++){
        System.out.println(i * i * i);
    }
    }
}

File: for loop/cube.cpp
/*Calculate the cube of all numbers from 1 to a given number*/

#include <iostream>
using namespace std;

int main(){
    int n,i;
    cout<<"Enter any number:";
    cin>>n;

    for(i=1; i<=n;i++){
        cout<<"Cube of "<<i<< " is " <<i * i * i<<endl;
    }
}